build.sh

```diff
@@ -70,6 +70,12 @@ if [ $? != 0 ]; then
     exit 1
 fi
 
+which lsb_release > /dev/null 2>&1
```

Comment:
I don't think we can do this: `lsb_release` is, I belive, a Linux-only thing (and maybe not even an every-Linux thing?). That means this commit would stop us running on non-Linu. What do we use `lsb_release` for? Can we find an alternative?

Reply:
We use it in the build script to determine how to install the Python libraries, this is the relevant code:

```sh
# Install rpy2 Python package.
DEB8_HACK="0"
if [ "$(lsb_release -si)" = "Debian" ]; then
    DEB_MAJOR_V=`lsb_release -sr | cut -c1`
    if [ "${DEB_MAJOR_V}" = "8" ] || [ "${DEB_MAJOR_V}" = "9" ]; then
        DEB8_HACK="1"
    fi
fi

if [ "${DEB8_HACK}" = "1" ]; then
    # Debian 8 or 9
    pip install --system --target "${PIP_TARGET_DIR}" "rpy2==2.8.5"
else
    pip install --target "${PIP_TARGET_DIR}" "rpy2==2.8.5"
fi
```

Reply:
We need something which isn't Linux specific-- we know people are using this on OS X, for example. At the very least, we would need to gate this i.e. say "if running on Debian, try this branch".

Reply:
I'm hoping the next commit has helped?